Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -191,9 +193,9 @@ FullDiagnostics::AddRZModesToDiags (int lev)
 
     // First index of m_all_field_functors[lev] where RZ modes are stored
     int icomp = m_all_field_functors[0].size();
-    const std::array<std::string, 3> coord {"r", "theta", "z"};
+    const std::array<std::string, 3> coord {"r", "t", "z"};
 
-    // Er, Etheta, Ez, Br, Btheta, Bz, jr, jtheta, jz
+    // Er, Et, Ez, Br, Bt, Bz, jr, jt, jz
     // Each of them being a multi-component multifab
     int n_new_fields = 9;
```

Comment:
Er, Et, Ez, Br, Bt, Bz, jr, jt, jz are automatically added - do we want more flexibility?

Reply:
automatic adding is good as equivalents to Cartesian.
Also in RZ, we want to allow to overwrite them via `<diag_name>.fields_to_plot`. That includes specifying more fields or less than the defaul

Note that adding not fields seems to cause problems at least with plotfiles at the moment: #2419 #1559

https://warpx.readthedocs.io/en/latest/usage/parameters.html#full-diagnostics